src/SDKs/Batch/Support/FileStaging/Batch.FileStaging/Batch.FileStaging.csproj

```diff
@@ -8,7 +8,7 @@ Visit our home page for more detail - http://azure.microsoft.com/services/batch/
 For technical overview, see http://azure.microsoft.com/documentation/articles/batch-technical-overview/.
 
 API reference can be found at http://go.microsoft.com/fwlink/?LinkId=717949.</Description>
-    <VersionPrefix>8.0.1</VersionPrefix>
```

Comment:
We recommend using `Version` instead of `VersionPrefix`, please fix wherever appropriate

Reply:
Same query regarding major version bump